modules/rosa-attaching-the-policy.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * rosa-adding-additional-constraints-for-ip-based-aws-role-assumption/rosa-attaching-the-policy.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-attaching-the-policy_{context}"]
+= Attaching the Identity-based IAM policy
+
+:context: rosa-attaching-the-policy
+
+toc::[]
+
+Once you have created an Identity-based IAM policy, you need to attach it to the relevant IAM users, groups, or roles in your AWS account. This enforces IP-based role assumption prevention for those entities.
```

Comment:
```suggestion
Once you have created an identity-based IAM policy, attach it to the relevant IAM users, groups, or roles in your AWS account to prevent IP-based role assumption for those entities.
```